runtime/near-evm-runner/Cargo.toml

```diff
@@ -34,6 +34,7 @@ ethereum-types = "0.8.0"
 ethabi = "11.0.0"
 
 near-vm-logic = { path = "../near-vm-logic" }
+near-primitives = { path = "../../core/primitives" }
```

Comment:
I am not sure about near-evm-runner, but I think it is still preferable to keep it slim and avoid dependency on near-primitives.